### ISSUE TITLE
Mycelium EIP155 ERROR

### ISSUE THREAD

**Issue by Kogut55** [mycelium-com/wallet-android#602]
Unfortunately the update does not fix the error, ETH can now be sent.  I still can't send ERC20 tokens, I have PAXi ETH tokens on my wallet, if I want to send PAX, there is still the same EIP155 error, THE UPDATE HAS NOT FIXED THE PROBLEM, THE TOKENS ARE BLOCKED, NO POSSIBILITY TO TRANSFER, EVEN IN INTERNAL ACCOUNTS

**Comment by alitohidi-dev** [comment]
Yes, I have the same problem and I can not transfer my ETH from my wallet
